guide/preamble.tex

```diff
@@ -291,6 +303,18 @@
     breaklinks=true
 ]{hyperref}  % TODO reuse infos
 
+% url
+\usepackage{relsize}
+\renewcommand*{\UrlFont}{\ttfamily\smaller\relax}
```

Comment:
@craice : weren't we going to typeset `\url`s with `\rmfamily` (like we [did](https://github.com/ITDP/the-online-brt-planning-guide/commit/13fa4770dd853fd78f92effe8fe804aafff7b457#diff-250e8ea0506a9f6f4a366f311e5c9ec0L24) with the HTML output)?